### ISSUE TITLE
Bytecode interpretation for Ebisp

### ISSUE THREAD

**Issue by voldyman** [tsoding/ebisp#25]
https://bernsteinbear.com/blog/bytecode-interpreters/

the article explains bytecode interpreters for lisp and can be used to improve ebisp.

**Comment by tekknolagi** [comment]
Huh, funny seeing my post in the wild. Did you ever get around to poking at it?

**Comment by rexim** [comment]
@tekknolagi nah, not really, sorry! This project is in deep hiatus for a while. :) Thanks for the post tho! :+1:
